solr/CHANGES.txt

```diff
@@ -271,6 +247,89 @@ Build
 
 * SOLR-15992: Globally forbid and exclude known bad dependencies (Kevin Risden)
 
+Docker
```

Comment:
I like that it has its own section now!
Granted I don't like CHANGES.txt at all anymore but while we have it...

Reply:
I think the "Build" and "Docker" sections make sense for the major release, but for 9.1 we fall back to the normal headings, since there will likely be few changes.